src/main/protobuf/extvm/msg.proto

```diff
@@ -102,20 +88,24 @@ message Blockhash {
 
 message CallResult {
   bytes returnData = 1;
-  bytes gasRemaining = 2;
-  bytes gasRefund = 3;
-  bool error = 4;
-  repeated ModifiedAccount modifiedAccounts = 5;
-  repeated bytes deletedAccounts = 6;
-  repeated bytes touchedAccounts = 7;
-  repeated LogEntry logs = 8;
+  // required for IELE VM
+  bytes returnCode = 2;
+  bytes gasRemaining = 3;
+  bytes gasRefund = 4;
+  bool error = 5;
```

Comment:
We could extract this from `returnCode` but this would require different logic per VM. We only need a simple `true` or `false` answer so retaining this field is probably simpler.